core-bundle/src/Resources/contao/templates/backend/be_main.html5

```diff
@@ -11,6 +11,7 @@
       <meta name="generator" content="Contao Open Source CMS">
       <meta name="viewport" content="width=device-width,initial-scale=1.0,shrink-to-fit=no">
       <meta name="referrer" content="origin">
+      <meta name="robots" content="noindex, nofollow">
```

Comment:
Why is it necessary to add this? The `/contao` route is exempt in the `robots.txt` file and no crawler will ever be able to see this template without being logged into the back end.

Reply:
You are logged into the back end if you authenticate with a front end user. There's no possibility to authenticate as a FE user statelessly at the moment.
And `/contao` is not except by the robots.txt apparently, otherwise Escargot would not crawl.

Reply:
> You are logged into the back end if you authenticate with a front end user.

What? That would be a major security hole!

Reply:
You misunderstood. I mean when you rebuild the search index from the back end, you are authenticated as a back end user and front end user at the same time. That's why.

Reply:
I see.

> And `/contao` is not except by the robots.txt apparently, otherwise Escargot would not crawl.

You have to delete the `web/robots.txt` file so the request triggers the new robots.txt route. It seems we forgot to remove the file when we merged #717. 🙈 

<img width="302" alt="" src="https://user-images.githubusercontent.com/1192057/71989422-eed5d980-3231-11ea-8937-8e6dfb5f5e6e.png">

Reply:
No, it was never there: https://github.com/contao/contao/pull/717/files#diff-a793700f1c87e1a3a0d01553e0c01f39
But it would make sense, yes. I'll check.

Reply:
Aaah, I see what you mean now. You're right. I had to remove the file and it's all good now.
However, I don't think deleting it is a good idea. The migration would need to copy the contents into the root pages so a possible previous configuration doesn't get lost.

Reply:
And I would still like to keep the meta tags because they aren't wrong :)